modules/openapi-generator/src/main/resources/erlang-server/handler.mustache

```diff
@@ -52,7 +52,7 @@ init(Req, {Operations, Module}) ->
     State = #state{operation_id = OperationID,
                    accept_callback = fun Module:accept_callback/4,
                    provide_callback = fun Module:provide_callback/4,
-                   api_key_handler = fun Module:authorize_api_key/2},
+                   api_key_callback = fun Module:api_key_callback/2},
```

Comment:
For a moment I feared this would be a breaking change, but actually, this is really a bugfix, the callback was declared as `api_key_callback/2` and dialyzer would complain, so the previous code would fail to bind to the right function and crash. Awesome find, thank you very much!